Header header
string init_agent_identifier
customPoint init_point
string[] agents 		        #the identifiers of the agents in the system 
customPoint[] my_domain
string ok_agent_identifier  		# for receive_ok method 
customPoint ok_value  			# for receive_ok method 
agentViewElement[] no_goods  		# for receieve_no_good method
bool init
bool ok
bool nogood
-------
float64 x
float64 y
float64 z
